src/Paket.Core/Logging.fs

```diff
@@ -2,82 +2,103 @@
 
 open System
 open System.IO
+open System.Diagnostics
 
+/// [omit]
 let mutable verbose = false
 
-let mutable logFile : string option = None
-let traceFunctions = System.Collections.Generic.List<_>()
+/// [omit]
+type Trace = {
+    Level: TraceLevel
+    Text: string
+    NewLine: bool }
 
-let setLogFile fileName =
-    let fi = FileInfo fileName
-    logFile <- Some fi.FullName
-    if fi.Exists then
-        fi.Delete()
-    else
-        if fi.Directory.Exists |> not then
-            fi.Directory.Create()
+/// [omit]
+let event = Event<Trace>()
 
-let inline traceToFile (text:string) =
-    match logFile with
-    | Some fileName -> try File.AppendAllLines(fileName,[text]) with | _ -> ()
-    | _ -> ()
+/// [omit]
+let subscribe callback = Observable.subscribe callback event.Publish
 
-let RegisterTraceFunction(traceFunction:Action<string>) =
```

Comment:
Who was using this? Will `Logging.subscribe` work for them or does this need to be added back (may be call `subscribeWithAction`). It would return a `IDisposable` instead of having `RemoveTraceFunction`.